generated/python/grpc-google-monitoring-v3/setup.py

```diff
@@ -0,0 +1,39 @@
+"""A setup module for the GRPC google-monitoring service.
+
+See:
+https://packaging.python.org/en/latest/distributing.html
+https://github.com/pypa/sampleproject
+"""
+
+import setuptools
+
+from setuptools import setup, find_packages
+
+install_requires = [
+  'oauth2client>=1.4.11',
+  'grpcio>=1.0.0',
+  'googleapis-common-protos[grpc]>=1.3.2'
+]
+
+setuptools.setup(
+  name='grpc-google-monitoring-v3',
+  version='0.9.0',
```

Comment:
Please regenerate with `0.10.x` series as with `gapic`. There are fewer changes for `gRPC`, but this will ensure consistency (including upper bounds for most dependencies above, and additional Trove data below).